tools/cuvslam_app/report_templates/report.html

```
@@ -99,6 +99,12 @@
     </tr>
     </table>
 
+{% if avg_tl_path or avg_rl_path %}
+    <h2>Average</h2>
+    {% if avg_tl_path %}<img src="{{ avg_tl_path }}">{% endif %}
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)